test/report_convert_to_pdf_test.py

```diff
@@ -0,0 +1,27 @@
+from os import remove, listdir
+from os.path import exists
+
+import pytest
+from docx import Document
+
+from src.report.report_utils import convert_docx_to_pdf
+
+
+@pytest.fixture(autouse=True)
+def file_cleanup():
```

Comment:
prefer use of TestCase and thus setUp and tearDown methods

Reply:
Changed all tests using fixtures to TestCase